### ISSUE TITLE
Display banner while contracts are syncing

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#1185]
### Problem

Currently there's no visual indicator that the app is loading anything when we're syncing contracts.

### Solution

While a specific loading design might need to be made for this at some point (cc @leihla), in the meantime we can get away with using the global BannerGeneral.vue to display a message to the users letting them know that we're loading data (see main.js):

```js
sbp('okTurtles.data/set', 'BANNER', this.$refs.bannerGeneral)
```

A message like, "Loading events from server..." should appear. Have a look in `state.js` for clues on how to close this issue.

**Comment by dotmacro** [comment]
> While a specific loading design might need to be made for this at some point 

[Skeleton loading designs for the app are on figma](https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN/Group-Income-2.0?node-id=5886%3A0), and were implemented in #948. 

Is this issue for when the user already has some data 100% loaded, but new data is syncing?

**Comment by taoeffect** [comment]
That issue was to display loading components/pages, and this issue is about displaying a loading banner for after all the pages are loaded, but the contract data isn't yet loaded.
